src/components/BookItem/BookItem.js

```diff
@@ -1,18 +1,60 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './_BookItem.scss';
+import { Pagination } from '../PagePagination/Pagination';
+
+let PageSize = 10;
 
 const BookItem = ({ item }) => {
+    const [currentPage, setCurrentPage] = useState(1);
+    const bookList = useMemo(() => {
+        const items = Array.isArray(item) ? item : [item];
+        const firstPageIndex = (currentPage - 1) * PageSize;
+        const lastPageIndex = firstPageIndex + PageSize;
+        return items.slice(firstPageIndex, lastPageIndex);
+    }, [currentPage, item])
+
     return (
-        <li className='book-item'>
-            <div>{item.title}</div>
-            <div>{item.language}</div>
-            <div>{item.ageRange}</div>
-            <div>{item.publishingYear}</div>
-            <div>{item.status}</div>
-            <div>{item.image}</div>
-            <div>{item.description}</div>
-            <div>{item.genre}</div>
-        </li>
+        <>
+        <table>
+            <thead>
+            <tr>
+                <th>Title</th>
+                <th>Language</th>
+                <th>Age Range</th>
+                <th>Publish Year</th>
+                <th>Status</th>
+                <th>Image</th>
+                <th>Description</th>
+                <th>Genre</th>
+            </tr>
+            </thead>
+            <tbody>
+            {bookList.map(item => {
+                return (
+                    <tr key={item.id}>
+                        <td>{item.title}</td>
+                        <td>{item.language}</td>
+                        <td>{item.ageRange}</td>
+                        <td>{item.publishingYear}</td>
+                        <td>{item.status}</td>
+                        <td>{item.image}</td>
+                        <td>{item.description}</td>
+                        <td>{item.genre}</td>
+                        <td>{item.title}</td>
+                    </tr>
+                )
+            }
+            )}
+        </tbody>
+        </table>
+            <Pagination
+            class="pagination-bar"
+            current={currentPage}
+            totalCount={Array.isArray(item) ? item.length : 1}
+            pageSize={PageSize}
+            onPageChange={page => setCurrentPage(page)}
+            />
+        </>
```

Comment:
Pagination should not be inside of BookItem here. BookItem should be inside of Pagination. I would revert the changes to BookItem and just place Pagination directly on a route to test.